src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace TEC\Tickets\Commerce;
+
+use TEC\Tickets\Settings as Tickets_Settings;
+use \tad_DI52_ServiceProvider;
+
+/**
+ * Class Payments_Tab
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce
+ */
+class Payments_Tab extends tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function register() {
+		$this->register_hooks();
```

Comment:
Providers are not accessible as a singleton, so we should make sure they are for unhooking purposes. 
```
$this->container->singleton( static::class, $this );
```